next/components/molecules/SearchPageNew/SearchCardComposed.tsx

```diff
@@ -25,12 +25,12 @@ const SearchCardComposed = ({ data, variant = 'default', tagText }: SearchCardCo
       {variant === 'default' && (
         <MLink
           className="group flex flex-row overflow-hidden"
-          // TODO fix href to accomodate cards with no slug
+          // TODO: fix href to accomodate cards with no slug
           href={isDefined(data.url) ? `${data.url}` : '#'}
         >
           <div className="flex w-full flex-row gap-6 px-6 py-4">
             <SearchCardComposed.InfoContainer className="gap-1.5">
-              <div className="flex flex-row flex-wrap items-center gap-x-3 gap-y-1.5 sm:flex-nowrap">
+              <div className="flex flex-row flex-wrap items-center gap-x-3 gap-y-1.5 lg:flex-nowrap">
```

Comment:
What is `lg:flex-nowrap` for? Does `lg:flex-col` do the same? Please use `flex-col` is possieble :)